### ISSUE TITLE
unhandled execption at "MCA HOWTOs" opening text link

### ISSUE THREAD

**Issue by realsobek** [vasole/pymca#863]
using Fedora Workstation 36 with PyMca-5.6.7-6.fc36.x86_64
1. PyMca -- Help -- MCA HOWTOs
2. scroll to bottom
3. click on "here" text link
4. error message is shown:
```
Unhandled execption
---
<class 'AttributeError'> 'QTextBrowser' object has no attribute 'raiseW'
Please report details
[Show Details...] [OK]
```

[Show Details...] shows:
`'QTextBrowser' object has no attribute 'raiseW'   File "<string>", line 1883, in setSource
`

**Comment by vasole** [comment]
Yep! I handled the `raise_` but I forgot about the very old `raiseW` alternative.

Thanks I will replace all the `.raiseW` by ` raise`

**Comment by vasole** [comment]
It should be closed by #864. Please re-open if needed.
